overlord/auth/auth.go

```diff
@@ -143,6 +145,9 @@ type NewUserData struct {
 	Macaroon string
```

Comment:
already landed but I would prefer this to be called NewUserParams

Reply:
Thanks for the feedback here! I added this to my TODO as a followup. It introduces to many changed files to do in this PR.

Reply:
https://github.com/snapcore/snapd/pull/12226